### PR TITLE
Radiobutton group issues

### PR DESCRIPTION
**Pull request by KRRISH96**
1. Label and Value accepted as properties in option prop.
2. Added Docs.
Issue #24 fixed

### CODE REVIEW

File: src/radioButtonGroup/index.js [hunk 1]
```diff
@@ -13,12 +13,14 @@ class RadioButtonGroup extends React.Component {
     };
   }
 
-  handleCheckListChange = (currentlyActive) => {
-    console.log(currentlyActive);
+  handleCheckListChange = (e, option) => {
```
Comment (prasanna1211) [inline src/radioButtonGroup/index.js]: This should be handleRadioBtnChange. This should change on every occurance

File: src/radioButtonGroup/index.js [hunk 2]
```diff
@@ -13,12 +13,14 @@ class RadioButtonGroup extends React.Component {
...
+    const { onClick } = this.props;
+    e.stopPropagation();
     this.setState({
-      currentlyActive,
+      currentlyActive: option.label,
     });
-  }
+    if (onClick) { onClick(option); }
```
Comment (prasanna1211) [inline src/radioButtonGroup/index.js]: Is it just a callback ? Then remove if case, just directly call onClick fn and set default props for onClick as empty fn.

File: src/radioButtonGroup/index.js [hunk 3]
```diff
@@ -33,7 +35,7 @@ class RadioButtonGroup extends React.Component {
         <div className={classNames}>
           {/* eslint-disable jsx-a11y/click-events-have-key-events */ }
           { /* eslint-disable jsx-a11y/no-static-element-interactions */ }
-          <div className={cx(styles['each-check'], { inline })} onClick={() => { this.handleCheckListChange(option.label); }}>
+          <div className={cx(styles['each-check'], { inline })} onClick={(e) => { this.handleCheckListChange(e, option); }}>
             <label className={cx(styles['customized-radio'], theme['customized-radio'])}>
               <label className={cx('inner', { checked: (option.label === this.state.currentlyActive) ? 'active' : '' })}>
```
Comment (prasanna1211) [inline src/radioButtonGroup/index.js]: extract the checked option before return statement. Basically make this more readable.

File: src/radioButtonGroup/readMe.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+| `onClick` | `Function` | &nbsp; | Callback function to be called when the radio button is clicked |
+| `inline` | `boolean` | `false` | Prop to define the alignment of radio button group |
+
+### Theme
+
+| Name     | Description|
+|:---------|:-----------|
+| `radio-group`   | Class used for the root element.|
+| `customized-radio`   | Class used for the radio button element.|
+| `each-check`   | Class used for the checkbox element.|
```
Comment (prasanna1211) [inline src/radioButtonGroup/readMe.md]: Re write the docs since it is mixed with check and radio. Every name should be related only to radio

**Review by prasanna1211 — CHANGES_REQUESTED** [review]
Need to do some changes before merging

File: src/radioButtonGroup/index.js (hunk 1 above)
Comment (KRRISH96) [reply src/radioButtonGroup/index.js]: Updated.

File: src/radioButtonGroup/index.js (hunk 2 above)
Comment (KRRISH96) [reply src/radioButtonGroup/index.js]: Removed if condition. Default props already added.

File: src/radioButtonGroup/index.js (hunk 3 above)
Comment (KRRISH96) [reply src/radioButtonGroup/index.js]: Extracted the checked option before return statement.

File: src/radioButtonGroup/readMe.md (hunk 4 above)
Comment (KRRISH96) [reply src/radioButtonGroup/readMe.md]: Docs typo fixed.

**Review by prasanna1211 — APPROVED** [review]

**Pull request merged**